av/video/codeccontext.pyx

```diff
@@ -162,3 +162,10 @@ cdef class VideoCodecContext(CodecContext):
     property coded_height:
         def __get__(self):
             return self.ptr.coded_height
+
+    property max_b_frames:
+        def __get__(self):
```

Comment:
Could we have a docstring explaining the use of this property please?

Does it make sense for decoding / encoding / both?